pkg/models/live.go

```diff
@@ -20,7 +20,7 @@ type ChannelHandler interface {
 // ChannelHandlerProvider -- this should be implemented by any core feature
 type ChannelHandlerProvider interface {
 	// This is called fast and often -- it must be synchrnozed
```

Comment:
Do you mind fixing this typo?
```suggestion
	// This is called fast and often -- it must be synchronized.
```

Reply:
Most of these comments are copy-pasted, but I can fix them, sure